atlp-rwanda/dynamites-ecomm-fe
# Fix: Navbar

### What does this PR do?
- This PR fixes the navbar such that when the user is not authenticated, it shows a login button

### How can this be manually tested?
- When user is not logged in, a login button should display in the navbar
- When user is logged in, their name and profile image should be displayed instead
- User can sign out using the signout button

### Screenshots
<img width="914" alt="zz" src="https://github.com/atlp-rwanda/dynamites-ecomm-fe/assets/109604986/7d02195f-9c9d-4e5b-ac04-2bd22451b896">
